pkg/ccl/testutilsccl/serverless_only.go

```diff
@@ -0,0 +1,8 @@
+package testutilsccl
+
+// ServerlessOnly is called in tests to mark them as testing functionality that
+// is Serverless specific. This is changed from a no-op to a test skip once
+func ServerlessOnly() {
+	// Uncomment in release branches that no longer support serverless.
+	// skip.WithIssue(t, 130441)
```

Comment:
Let's not use `skip.WithIssue()` as that implies that you the tests are broken due to a bug described in the given GitHub issue, and you are going to un-skip the tests when the issue is closed. That is not the case here.

Reply:
I think in that case I need to add a new skip function since all other skips are conditional. Maybe something like `skip.Unsupported()`

Reply:
`skip.IgnoreLint()` is what is typically used (poorly named, but it's what is used for unconditional skips)

Reply:
Fixed